cmd/utils.go

```diff
@@ -9,16 +9,16 @@ import (
 	"github.com/bitrise-core/bitrise-init/utility"
 )
 
-// IDEType ...
-type IDEType string
+// ProjectType ...
```

Comment:
Please take the time to describe the exported package elements.

Reply:
Fixed in:
https://github.com/bitrise-tools/codesigndoc/pull/95/commits/3198ad523fc879f992f0dd07629cb8315282961b

There is no need for the ProjectType to be exported => Changed it to private.